aiodebug/hang_inspection.py

```diff
@@ -0,0 +1,98 @@
+from typing import List, Optional
+import sys
+import traceback
+import asyncio
+import time
+import os
+import threading
+import datetime
+
+from pygments import highlight
+from pygments.lexers import PythonLexer
+from pygments.formatters import HtmlFormatter
```

Comment:
Can we get rid of these and dump plain text traces? We don't really need syntax highlighting, and this library should be lightweight with very few dependencies.